File: contest130/main.py
class Solution:
    def prefixesDivBy5( self, bits: List[int] ) -> List[bool]:
        ans = []
        N = 0
        for bit in bits:
            N = (( N << 1 ) + bit ) % 5
            ans.append( N == 0 )
        return ans

File: contest130/java/src/Main.java
import java.util.List;
import java.util.ArrayList;

public class Main {

    public static List<Boolean> prefixesDivBy5( int[] bits ){
        List< Boolean > ans = new ArrayList<>();
        int N = 0;
        for( int bit: bits ){
            N <<= 1;
            N |= bit;
            N %= 5;
            ans.add( N == 0 );
        }
        return ans;
    }

    public static void main(String[] args) {
        System.out.println("Hello World!");
    }
}
